localization/english/bookmark/bookmark_603_l_english.yml

```diff
@@ -1,11 +1,27 @@
 
 
  #Arthas Menethil
  arthas_menethil_character:0 "Prince Arthas"
- arthas_menethil_character_desc:0 "PH"
+ arthas_menethil_character_desc:0 "Only male heir to King Terenas II of Lordaeron, Arthas has been prepared for his future duty by the best tutor Lordaeron can offer, Lord Uther Lightbringer of the Silver Hand, who trained the young prince as a paladin. Skilled and devoted, Arthas is no king yet, serving his father as commander and vassal, eager to prove himself and deeply devoted to his people. King Terenas is an old, wise, and cautious man, it is up to prince Arthas to lead by example and action in the fights to come.\n\nWill Arthas be the ruler that Lordaeron needs or will his recklessness do more bad than good?"
+
+ #Dar'khan Drathir
+ darkhan_character:0 "Dar'Khan"
+ darkhan_character_desc:0 "A magister, mage-lord of Quel'Thalas, Dar'Khan served his liege, king Anasterian, as well as the Alliance against the Trolls and Orcs during the second war, and he did so with great skills and efficiency. The mage, however, is also an expert on the magic of the Sunwell, Quel'Thalas's main source of power, as well as in the arts of magical buildings and esoteric studies. Despite all his accomplishments, Dar'Khan lacks the recognition he deserves and aspires to more than what his ungrateful peers accept to grant him, yet he knows he is capable to achieve greatness and carve his mark in history. Is Dar'Khan promised to a great destiny or will he be forgotten like so many before him that weren't ambitious enough ."
+
+ #Anasterian Sunstrider
+ anasterian_character:0 "High King Anasterian"
+ anasterian_character_desc:0 "For centuries already have Anasterian ruled over Quel'Thalas as its king, heir to the line of Dath'remar. Wise but proud, the old king values independance and tranquility and as such withdrawed entirely from the Alliance after its victory against the Horde during the second war. With the orcs and trolls repelled from his frontiers, the king is confident that his realm's prosperity and magical superiority will assure his people's safety in the haven that are the thalassian forests ; numerous elves, including crown-prince Kael'Thas, however, remain sympathethic to the Alliance and believe it is the elves's duty to help their former allies in these difficult, changing times. Even the secluded elves can't ignore the events of the world forever. What destiny awaits the elven king and his people ? To prosper secluded in their magical kingdom ? To unite with their formers allies against a darker threat ? To fall into oblivion alone...Or perhaps to take an even greater place in the world ? "
```

Comment:
```suggestion
 anasterian_character_desc:0 "For centuries already have Anasterian ruled over Quel'Thalas as its king, heir to the line of Dath'remar. Wise but proud, the old king values independence and tranquility and as such withdraws entirely from the Alliance after its victory against the Horde during the second war. With the orcs and trolls repelled from his frontiers, the king is confident that his realm's prosperity and magical superiority will assure his people's safety in the haven that are the thalassian forests; numerous elves, including crown-prince Kael'Thas, however, remain sympathetic to the Alliance and believe it is the elves' duty to help their former allies in these difficult, changing times. Even the secluded elves can't ignore the events of the world forever. What destiny awaits the elven king and his people? To prosper secluded in their magical kingdom? To unite with their former allies against a darker threat? To fall into oblivion alone...Or perhaps to take an even greater place in the world? "
```

Reply:
```suggestion
 anasterian_character_desc:0 "For centuries, Anasterian Sunstrider has ruled over Quel'Thalas as its king, heir to the line of Dath'remar. Wise but proud, the old king values independence and tranquility and as such withdrew entirely from the Alliance after victory was acquired against the Horde during the Second War. With the orcs and trolls repelled from his frontiers, Anasterian is confident that his kingdom's prosperity and magical superiority will assure his people's safety and happiness in the haven that are the thalassian forests; numerous elves, including crown-prince Kael'Thas, however, remain sympathetic to the Alliance and believe it is the elves' duty to help their former allies in these difficult, changing times. Even the secluded elves can't ignore the events of the world forever.\n\nWhat destiny awaits the elven king and his people? To prosper secluded in their magical kingdom? To unite with their former allies against a darker threat? To fall into oblivion alone? Or perhaps to take an even greater place in the world?"
```

Reply:
^ another 3-week-old suggestion with deleted unnecessary spaces and added line breaks...